RebootServer.py

```diff
@@ -154,14 +167,16 @@ def RunServer(miners):
             sleep(300)
 
             print("Starting regular checks at %s..." % getTime(), flush=True)
-            for miner in miners.keys():
-                if not miners[miner].healthCheck():
-                    print("Miner %s wasn't responding, so we're rebooting it!" % miners[miner].name, flush=True)
-                    miners[miner].reboot()
-                else:
-                    print("miner %s's health check passed!" % miners[miner].name, flush=True)
-                    pass
-                # End if/else block
+            if connection_check==true:
```

Comment:
What happens if the connection check is false? Shouldn't we print a message to tell us that connectivity was lost?

Just FYI, it's `True` or `False`; capitalization matters here.

One last thing; in Python, you can test for boolean (T/F) values by simply saying `if connection_check:`, as it implies `if connection_check != True:`